Linux/Car_Client/chronos.cpp

```diff
@@ -305,3 +307,67 @@ void Chronos::processMtsp(chronos_mtsp mtsp)
                     mSypmLast.sync_point - mSypmLast.stop_time;
     }
 }
+
+void Chronos::processRcmm(chronos_rcmm rcmm)
+{
+    qDebug() << "RCMM RX";
+
+    // TODO: Check object state? Maneouver command?
```

Comment:
If you have the time, please implement the object state checking now. We already have a bit of experience with the RC car behaving unexpectedly, better not to add more.

Reply:
RCMM should only be allowed in the REMOTECTRL state.

Reply:
You can ignore the manoeuvre command